code/modules/surgery/organs/organ_internal.dm

```diff
@@ -204,32 +204,37 @@
 	else
 		var/obj/item/organ/lungs/L = getorganslot(ORGAN_SLOT_LUNGS)
 		if(!L)
-			L = new()
-			L.Insert(src)
+			if(!(L.organ_flags & ORGAN_UNREMOVABLE))
+				L = new()
+				L.Insert(src)
 		L.setOrganDamage(0)
 
 		var/obj/item/organ/heart/H = getorganslot(ORGAN_SLOT_HEART)
 		if(!H)
-			H = new()
-			H.Insert(src)
+			if(!(H.organ_flags & ORGAN_UNREMOVABLE))
+				H = new()
+				H.Insert(src)
 		H.setOrganDamage(0)
 
 		var/obj/item/organ/tongue/T = getorganslot(ORGAN_SLOT_TONGUE)
 		if(!T)
-			T = new()
-			T.Insert(src)
+			if(!(T.organ_flags & ORGAN_UNREMOVABLE))
+				T = new()
+				T.Insert(src)
 		T.setOrganDamage(0)
 
 		var/obj/item/organ/eyes/eyes = getorganslot(ORGAN_SLOT_EYES)
 		if(!eyes)
-			eyes = new()
-			eyes.Insert(src)
+			if(!(eyes.organ_flags & ORGAN_UNREMOVABLE))
+				eyes = new()
+				eyes.Insert(src)
 		eyes.setOrganDamage(0)
 
 		var/obj/item/organ/ears/ears = getorganslot(ORGAN_SLOT_EARS)
 		if(!ears)
-			ears = new()
-			ears.Insert(src)
+			if(!(ears.organ_flags & ORGAN_UNREMOVABLE))
+				ears = new()
+				ears.Insert(src)
```

Comment:
the organ itself doesn't exist at this point, so this would either runtime or have undefined behavior. Is it actually needed?

Reply:
Right, didn't think about that. Just to make 100% sure, I'll test if it somehow breaks if I remove it.

Reply:
It didn't break, I'll revert it now.